<ui version="4.0" >
 <class>Settings</class>
 <widget class="QDialog" name="Settings" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>249</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Global Settings</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Maximum memory for undo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="SpinBoxUndo" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>90</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="suffix" >
        <string> Mb</string>
       </property>
       <property name="maximum" >
        <number>100000000</number>
       </property>
       <property name="value" >
        <number>512</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Undo Timer Interval</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="SpinBoxUndoTimer" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>90</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="suffix" >
        <string> seconds</string>
       </property>
       <property name="minimum" >
        <number>1</number>
       </property>
       <property name="maximum" >
        <number>100</number>
       </property>
       <property name="value" >
        <number>4</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>230</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Maximum memory for cache</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="SpinBoxCache" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>90</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="suffix" >
        <string> Mb</string>
       </property>
       <property name="maximum" >
        <number>100000000</number>
       </property>
       <property name="value" >
        <number>512</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="BoxBackCache" >
     <property name="layoutDirection" >
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text" >
      <string>Background caching</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="BoxRenderCache" >
     <property name="layoutDirection" >
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text" >
      <string>Tune Cache for Rendering</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>230</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_8" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>File Compression</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="SliderFileCompression" >
       <property name="maximum" >
        <number>2</number>
       </property>
       <property name="pageStep" >
        <number>1</number>
       </property>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition" >
        <enum>QSlider::TicksBothSides</enum>
       </property>
       <property name="tickInterval" >
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="LabelFileCompression" >
       <property name="minimumSize" >
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>Off</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QPushButton" name="RecompressSourceFiles" >
       <property name="text" >
        <string>(Re)compress source files now</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="RecompressFiles" >
       <property name="text" >
        <string>(Re)compress working files now</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_9" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>Cache Compression:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="SliderCacheCompression" >
       <property name="maximum" >
        <number>2</number>
       </property>
       <property name="pageStep" >
        <number>1</number>
       </property>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition" >
        <enum>QSlider::TicksBothSides</enum>
       </property>
       <property name="tickInterval" >
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="LabelCacheCompression" >
       <property name="minimumSize" >
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>Off</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame" >
     <property name="minimumSize" >
      <size>
       <width>16</width>
       <height>24</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>0</number>
      </property>
      <property name="margin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>Autosave Frequency</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="AutoSave" >
        <property name="maximumSize" >
         <size>
          <width>90</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="suffix" >
         <string> minutes</string>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="value" >
         <number>5</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>230</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton" >
     <property name="text" >
      <string>Clear Recent Files List</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>Changes will take effect on opening a dataset</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>230</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
